pkg/segment/writer/packer.go

```diff
@@ -1556,6 +1556,20 @@ func addSegStatsStrIngestion(segstats map[string]*SegStats, cname string, valByt
 		segstats[cname] = stats
 	}
 
+	floatVal, err := strconv.ParseFloat(string(valBytes), 64)
+	if err == nil {
+		if !stats.IsNumeric {
+			stats.IsNumeric = true
+		}
+		addSegStatsNums(segstats, cname, SS_FLOAT64, 0, 0, floatVal, valBytes)
+		return
```

Comment:
Wouldn't we only want to set `stats.IsNumeric = true` if we get here and we're not sure if it's numeric or not (i.e., this is the first entry in `stats`)? With the current way I think you could add a string stat, get `IsNumeric = false`, then add a numeric stat here and set `IsNumeric = true` but then you'd probably (?) have issues later? Or do we always want to use numeric stats if we have any?

Reply:
We are currently not adding any stringStat in sst.